drivers/driver-test/src/commonMain/kotlin/com/squareup/sqldelight/driver/test/QueryTest.kt

```diff
@@ -136,10 +177,13 @@ class QueryTest {
   private fun testDataQuery(): Query<TestData> {
     return object : Query<TestData>(copyOnWriteList(), mapper) {
       override fun createStatement(): SqlPreparedStatement {
-        return connection.prepareStatement(0, "SELECT * FROM test", SELECT, 0)
+        return database.getConnection().prepareStatement(0, "SELECT * FROM test",
+            SqlPreparedStatement.Type.SELECT, 0)
       }
     }
   }
 
   private data class TestData(val id: Long, val value: String)
 }
+
+private fun Atomic<Int>.increment() = set(get() + 1)
```

Comment:
Maybe leave a comment that this isn't actually atomic but there's no concurrency